doc/devel/documenting_mpl.rst

```diff
@@ -88,16 +93,85 @@ errors.  To unset it, use
 
 You can use the ``O`` variable to set additional options:
 
-* ``make O=-j4 html`` runs a parallel build with 4 processes.
-* ``make O=-Dplot_formats=png:100 html`` saves figures in low resolution.
-* ``make O=-Dplot_gallery=0 html`` skips the gallery build.
+.. code-block:: sh
+    
+    #runs a parallel build with 4 processes.
+    make O=-j4 html 
+    
+    #saves figures in low resolution.
+    make O=-Dplot_formats=png:100 html 
+    
+    #builds the gallery without executing the scripts
+    make O=-Dplot_gallery=0 html 
+
+    #Multiple options can be combined using e.g.    
+    make O='-j4 -Dplot_gallery=0' html
+    
+Windows
+~~~~~~~
+.. note::
+
+    It is currently not possible to build the gallery on windows due to a 
+    problem in sphinx-gallery that should be fixed in version 0.1.14. A 
+    workaround is to not build the gallery by commenting out 
+    ``sphinx_gallery.gen_gallery`` in ``extensions`` in the ``conf.py`` file 
+    or to delete the code in or delete the  
+    ``examples\userdemo\pgf_preamble_sgskip.py`` file.
```

Comment:
This seems kind of a strange workaround. I'd say the workaround is to wait a few more days until 0.1.14 is finally released or install the current development version of sphinx-gallery.